services/config_service.rb

```diff
@@ -11,8 +11,10 @@ class ConfigService
     attr_accessor :project_service
     attr_accessor :ci_user
     attr_accessor :active_code_hosting_services # dictionary of active_code_hosting_service_key to CodeHosting
+    attr_accessor :clone_user_provider_credential
 
-    def initialize(project_service: FastlaneCI::Services.project_service, ci_user: nil)
+    def initialize(project_service: FastlaneCI::Services.project_service, ci_user: nil, clone_user_provider_credential:)
+      @clone_user_provider_credential = clone_user_provider_credential
```

Comment:
we can use `self.<variable>` = for our initialization now. It's best to standardize on something, and it seems to be the most popular (CC @KrauseFx who mentioned this a while back to me, I think)

Reply:
Yeah, that's cool with me - I'll make a follow-up PR to remove other instances in the codebase where we use `@variable =` too.